plugins/gcloud/README.md

```diff
@@ -0,0 +1,25 @@
+# gcloud
+
+This plugin provides completion support for the
+[Google Cloud SDK CLI](https://cloud.google.com/sdk/gcloud/).
+Based on [gcloud plugin for oh-my-zsh](https://github.com/ohmyzsh/ohmyzsh/tree/master/plugins/gcloud).
+
+To use it, add `gcloud` to the plugins array in your bashrc file.
+
+```bash
+plugins=(... gcloud)
+```
+
+It relies on you having installed the SDK using one of the supported options
+listed [here](https://cloud.google.com/sdk/install).
+
+## Plugin Options
+
+* Set `CLOUDSDK_HOME` in your `bashrc` file before you load oh-my-bash if you have

```

Comment:
As far as I search for the string `CLOUDSDK_HOME`, the variable seems to be specific to the oh-my-zsh plugin. For the shell variables used by Oh My Bash, we are gradually applying the naming convention discussed in #280. In particular, we would like to adopt the naming convention from the beginning to avoid unnecessary later migrations. For the present purpose, for example, `OMB_PLUGIN_GCLOUD_HOME` follows the naming convention.
```suggestion
* Set `OMB_PLUGIN_GCLOUD_HOME` in your `bashrc` file before you load oh-my-bash if you have
```

(The other occurrences of the name `CLOUDSDK_HOME` also need to be updated.)

Reply:
You are correct! I did assume `CLOUDSDK_HOME` was an internal variable to `gcloud`. Yet, perusing the source code of the installation script, it can be seen they define it as `CLOUDSDK_ROOT_DIR` and it's used only as a local variable in the installation phase. I think sticking to the namespacing it's a very sensible choice!